src/components/projects/teams/AddTeamMemberDialog.tsx

```diff
@@ -32,19 +32,23 @@ const AddTeamMemberDialog = ({
   addMembers,
 }: Props) => {
   const [searchText, setSearchText] = useState("")
+  const [loading, setLoading] = useState(false)
+
   const [searchResults, setSearchResults] = useState<User[]>([])
   const [selectedUsers, setSelectedUsers] = useState<IMultiSelectOptions[]>([])
 
-  const [debouncedSearchText] = useDebounceValue(searchText, 100)
+  const [debouncedSearchText] = useDebounceValue(searchText, 150)
 
   const onAddMembers = useCallback(async () => {
     try {
+      setLoading(true)
       await addMembers(selectedUsers.map((user) => user.value.toString()))
-      onOpenChange(false)
     } catch (error) {
       console.error("Error adding team members", error)
```

Comment:
would be cute to add a little toast on error too

Reply:
trueeeeee